src/app/pages/jails/storages/storage-list/storage-list.component.ts

```diff
@@ -92,7 +91,7 @@ export class StorageListComponent {
       ]
     ]).subscribe((res) => {
       if (res[0] && res[0].state == 'up') {
-        this.snackBar.open(this.jailId + " should not be running when deleting a mountpoint.", 'close', { duration: 5000 });
+        this.dialog.Info(T('Delete Mountpoint'), "<i>" + this.jailId + T("</i> should not be running when deleting a mountpoint."), '500px', 'info', true);
```

Comment:
```suggestion
        this.dialog.Info(T('Delete Mountpoint'), "<i>" + this.jailId + T("</i> cannot be running when deleting a mountpoint."), '500px', 'info', true);
```
Is the dialog offering a suggestion or communicating an error?